doc/user/layouts/shortcodes/postgres-direct/create-an-ingestion-cluster.html

```diff
@@ -0,0 +1,11 @@
+In Materialize, a [cluster](/get-started/key-concepts/#clusters) is an **isolated environment**, similar to a virtual warehouse in Snowflake. Within a cluster, you have [replicas](/get-started/key-concepts/#cluster-replicas), which are the **physical resources** for doing computational work. Clusters are completely isolated from each other, so replicas can be sized based on the specific task of the cluster, whether that is ingesting data from a source, computing always-up-to-date query results, serving results to clients, or a combination.
+
+In this case, you'll create 1 new cluster containing 1 replica for ingesting source data from your PostgreSQL database.
+
+1. In the `psql` shell connected to Materialize, use the [`CREATE CLUSTER`](/sql/create-cluster/) command to create the new cluster and replica:
+
+    ```sql
+    CREATE CLUSTER ingest_postgres REPLICAS (r1 (SIZE = 'medium'));
+    ```
+
+    We recommend starting with a `medium` [size](/sql/create-cluster-replica/#sizes) replica or larger. This will help Materialize more quickly take an initial snapshot of the tables in your publication. Once the snapshot is finished, you'll right-size the cluster replica.
```

Comment:
```suggestion
    We recommend starting with a `medium` [size](/sql/create-cluster-replica/#sizes) replica or larger. This will help Materialize more quickly process the initial snapshot of the tables in your publication. Once the snapshot is finished, you'll right-size the cluster replica.
```

Reply:
Done.